out/production/online-shop-cristianfodor96/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Remove the following folders from the project root: `.idea`, `out`. These are IDE generated folders that should not be committed.  They were generated in this folder because you opened `online-shop-cristianfolder96` in IntelliJ. There are 2 options: 
- add a `.gitignore` file in the project root (`online-shop-cristianfolder96`) to ignore `.idea` and `out` next time you open this folder in IntelliJ and you commit your work
- always open the `shop` folder in IntelliJ (which is a subfolder of `online-shop-cristianfolder96`). It already contains a `.gitignore` file, so the generated files and folders will not be commited